drivers/bridge/bridge.go

```diff
@@ -144,7 +144,7 @@ func newDriver() *driver {
 }
 
 // Init registers a new instance of bridge driver
-func Init(dc driverapi.DriverCallback, config map[string]interface{}) error {
+func Init(dc driverapi.DriverCallback, ps interface{}, config map[string]interface{}) error {
```

Comment:
got it. Can you instead use the above `DriverCallback` for this purpose ?
You can return the plugingetter as a GetPluginGetter API as part of DriverCallback. 
BTW, `Controller` implements `DriverCallback` and hence you can get this done cleanly without having to change the `Init` API signature.